vm/src/lib.rs

```diff
@@ -66,3 +66,8 @@ mod vm;
 pub use self::exceptions::print_exception;
 pub use self::vm::VirtualMachine;
 pub use rustpython_compiler::*;
+
+#[doc(hidden)]
+pub mod __exports {
```

Comment:
I think the same can be achieved by using `extern crate bincode`. Cargo will make sure that both the vm and the wasm crate use the same version of the bincode crate, so no need to import the bincode crate via the `rustpython_vm`.

Reply:
I did this here because this pattern is used by other macros in the ecosystem, here are a few examples:
- https://docs.rs/serde/1.0.92/src/serde/lib.rs.html#236-242
- https://docs.rs/wasm-bindgen/0.2.47/src/wasm_bindgen/lib.rs.html#801
- https://docs.rs/darling/0.9.0/src/darling/lib.rs.html#84-95